src/components/TodoItem/TodoItem.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import cn from 'classnames';
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  todo: Todo
+};
+
+export const TodoItem: React.FC<Props> = ({ todo }) => {
+  return (
```

Comment:
```suggestion
export const TodoItem: React.FC<Props> = ({ todo }) => (
```

Reply:
I can't delete 'return' as i add Destructuring of todo.